test_cleanup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+rm -rf /usr/local/go1.15.x || true
```

Comment:
Is || true needed?

Reply:
Turns out that -f takes care of errors so in this case it actually isn't